src/include/verify.h

```diff
@@ -0,0 +1,27 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WT_VERIFY_INFO -- A structure to hold all the information related to a verify operation.
+ */
+struct __wt_verify_info {
+    WT_SESSION_IMPL *session;
+
+    const char *tag;           /* Identifier included in error messages */
+    const WT_PAGE_HEADER *dsk; /* The disk header for the page being verified */
+    WT_ADDR *item_addr;        /* An item representing a page entry being verified */
```

Comment:
Why are we referring to an addr as an item here? I feel like it is confusing with respect to WT_ITEM. The addr references a page so would a better name be `page_addr`?